### DISCUSSION TITLE
Αρχείο και γλώσσα παρουσίασης ερευνητικού άρθρου

### DISCUSSION THREAD

**Discussion by epidrome** [sdy60/syllabus#19] [Q&A] [answered]
Συχνές απορίες για τις παρουσιάσεις ερευνητικών άρθρων

Για τις δηλώσεις παρουσίασης θα πρέπει να κάνετε δήλωση στο σχετικό θέμα #21

**Comment by afoscolou** [comment]
Θα ήθελα να ρωτήσω αν θα πρέπει να κάνουμε drag & drop την ίδια την παρουσίαση, δηλ. το αρχείο του PowerPoint ή κάποιο pdf) ή θέλετε μόνο να σας ενημερώσουμε ότι προτιθέμεθα να παρουσιάσουμε επισυνάπτοντας μόνο τον τίτλο. Σκέφτομαι ότι, αν επισυνάψουμε την παρουσίαση και την έχουν δει από πριν οι περισσότεροι, χάνεται μεγάλο μέρος από το ενδιαφέρον του κοινού...

**Reply by epidrome** [reply]
σωστό σκεπτικό με την έννοια ότι έχουμε αγωνία, εξίσου σωστό νομίζω πως είναι το σκεπτικό να γίνει μια λεπτομερής δήλωση την προηγούμενη μέρα ώστε να είναι οι συνάδελφοι διαβασμένοι και να γίνει και σχολιασμός, ένα επιστημονικό άρθρο δεν διαβάζεται σε μερικά λεπτά.

**Comment by sotkot** [comment, marked as answer]
Νομίζω ότι είναι πολύ εύστοχος ο προβληματισμός της κυρίας Φωσκόλου.

**Reply by epidrome** [reply]
πράγματι όλοι οι προβληματισμοί είναι εύστοχοι και για αυτό έγινε νέα ανάρτηση με περισσότερες πληροφορίες για την δήλωση της πρόθεσης παρουσίασης, ώστε να κρατήσουμε χωριστά τις απορίες από τις δηλώσεις.

**Comment by afoscolou** [comment]
Σχετικά με την παρουσίαση, θα ήθελα να ρωτήσω επίσης αν υπάρχει κάποιος περιορισμός στη γλώσσα που θα χρησιμοποιήσουμε για το κείμενο των διαφανειών, αν δηλαδή θα πρέπει να είναι στα Ελληνικά ή μπορεί και να είναι στα Αγγλικά.  Τα Αγγλικά διευκολύνουν επειδή η ορολογία πολλών εννοιών είναι πιο κομψή σε αυτή τη γλώσσα παρά ως μια μεταφρασμένη λέξη.

**Reply by epidrome** [reply]
συμφωνώ και εγώ προτιμώ τα αγγλικά αρκεί να μην έχει κάποιος άλλος πρόβλημα

**Reply by afoscolou** [reply]
Άρα υπάρχει ελεύθερη επιλογή για την γλώσσα του κειμένου των διαφανειών.  Προφανώς μιλούσα για τη δική μου παρουσίαση καθ' ότι κάποιος συνάδελφος μπορεί ήδη να την έχει ετοιμάσει στα Ελληνικά.

**Reply by sotkot** [reply]
Συμφωνώ. Ας υπάρχει η δυνατότητα να γίνει η παρουσίαση είτε στα Αγγλικά είτε στα Ελληνικά. Μπορεί κάποιος να την έχει ήδη ολοκληρώσει.
